tests/integration/test_distributed_load_balancing/test.py

```diff
@@ -0,0 +1,114 @@
+# pylint: disable=unused-argument
+# pylint: disable=redefined-outer-name
+# pylint: disable=line-too-long
+
+import uuid
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+n1 = cluster.add_instance('n1', main_configs=['configs/remote_servers.xml'])
+n2 = cluster.add_instance('n2', main_configs=['configs/remote_servers.xml'])
+n3 = cluster.add_instance('n3', main_configs=['configs/remote_servers.xml'])
+
+nodes = len(cluster.instances)
+queries = nodes*5
+
+def create_tables():
+    for n in cluster.instances.values():
+        n.query('DROP TABLE IF EXISTS data')
+        n.query('DROP TABLE IF EXISTS dist')
+        n.query('CREATE TABLE data (key Int) Engine=Memory()')
+        n.query("""
+        CREATE TABLE dist AS data
+        Engine=Distributed(
+            replicas_cluster,
+            currentDatabase(),
+            data)
+        """.format())
+
+def make_uuid():
+    return uuid.uuid4().hex
+
+@pytest.fixture(scope='module', autouse=True)
+def start_cluster():
+    try:
+        cluster.start()
+        create_tables()
```

Comment:
@alexey-milovidov actually there is a very simple fix/quirk for the test flap, execute `SYSTEM RELOAD CONFIG` (on each node) just after start (here), this will avoid extra reload after server startup after reload_interval=2 seconds (don't want to create PR for oneline fix, let me know if you will need a patch)

Reply:
Yes, I need it.

PS. I did not have enough attention to this problem. If you think it will fix the test - let's do.

Reply:
Added to the #11669 (since it uses that test to cover functionality) - https://github.com/ClickHouse/ClickHouse/pull/11669/commits/bd45592539a2351c1547c5e3ef8951bcb13dd1a0#diff-a1856224acda58680decd89ce9251a00R21

If you don't bother looking at PR, just cherry pick it (I will rebase PR later)

Reply:
integration tests passed - bd45592539a2351c1547c5e3ef8951bcb13dd1a0